airbyte-integrations/connectors/destination-tidb/README.md

```diff
@@ -0,0 +1,68 @@
+# Destination TiDB
+
+This is the repository for the TiDB destination connector in Java.
+For information about how to use this connector within Airbyte, see [the User Documentation](https://docs.airbyte.io/integrations/destinations/ti-db).
```

Comment:
`ti-db`?

Reply:
@zhangyangyu Thanks for your review. I will correct it.